method/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Set the install prefix (you can modify this to another path if needed)
+INSTALL_PREFIX="$HOME/.local"
```

Comment:
This path is not what `.github/workflows/memleaks.yml` expects, time to update the `memleaks.yml` action?

Reply:
memleaks updated and issue fixed as seen in [this run](https://github.com/imallona/yamet/actions/runs/12330971674/job/34417039757?pr=9).